### DISCUSSION TITLE
What country does 🐯🍺 originally come from?

### DISCUSSION THREAD

**Discussion by ParanoidUser** [ParanoidUser/yolo#1262] [Galaxy Trivia] [answered]
**Bonus:** What beverage is served at a "kaffeeklatsch"?

**Comment by tomerf** [comment, marked as answer]
Tiger Beer is a Singaporean brand of beer. Launched in 1932, Tiger beer became Singapore's first locally brewed beer.

Bonus: kaffeeklatsch is an informal social gathering at which coffee is served.
